examples/helloWorld/README.md

```diff
@@ -402,7 +407,7 @@ Run kustomize again to see the new names:
 <!-- @grepStagingName @test -->
 ```
 kustomize build $OVERLAYS/staging |\
-    grep -B 8 -A 1 staging-the-map
+    kustomize build $OVERLAYS/staging | grep -B 2 -A 3 kiwi
```

Comment:
The original command is correct.  The point is to show the new hash suffix of the configmap name

Reply:
I think that is the point in line 380 - this particular spot should show the changed fruit in the greeting?

The new names are checked on below.

Reply:
The change from pineapple to kiwi means the content of the configMap changes which means the hash of that content changes, which means the suffix after _staging-the-map_ changes, which is what this command is meant to show - and yes, compare to what happened to the name of the configmap up above.

Perhaps change the description to:
 > Run kustomize again to see the new suffix of the configMap:

You're right that this is explicitly tested below in the grep for khk45 (which is how this markdown is tested), it's just that this command here prints it to the screen more obviously in context with the rest of the output.

Instead of replacing this command, you could precede it with a command that grepped for kiwi.

Regardless, one cannot pipe kustomize output into kustomize input, since kustomize doesn't take input from stdin.

Reply:
yes please, show the kiwi explicitly too - just needs a distinct kustomize command

Reply:
I see my confusion now. Will add the kiwi and make it clear. :)

Would you mind, when you have a moment, to help me learn how this markdown is tested?